README.md

```diff
@@ -25,10 +25,17 @@ The ASU Unity Design System packages are published to GitHub's package registry.
 5. Test installing packages using yarn or npm inside of another NPM project:
 ```yarn add @asu/design-tokens```
 
-### TROUBLESHOOTING INSTALLATION ERRORS
+### TROUBLESHOOTING ERRORS
 
 If you get errors having to do with yarn not being able to find a package on the registry, try running ```yarn config list``` at the project root and look for the ```registry:``` key under yarn config. If it is not set to ```https://registry.yarnpkg.com``` then run ```yarn config delete registry``` and recheck config.
 
+For custom compilation and sub-themes using the `unity-bootstrap-theme` package, make sure your bundling/compilation tool is compatible with the scss. For example, if you are usiing Gulp for scss compilation, you will need to handle the new module imports used in our webpack configuration.
+
+In the old bs4 theme we would import some bootstrap scss with `@import "../../node_modules/bootstrap/scss/root";`
+In the new theme it is imported directly with  `@import "bootstrap/scss/functions";`
+
+[This slack thread](https://asu-tech.slack.com/archives/C01ABUVU8LQ/p1695234506292239) has some more information about this issue and using Gulp. *Must be a member of the ASU Community Slack workspace to view.*
+
```

Comment:
I hate to be the typo guy, but....
Line 32 `usiing` extra i
Line 32, capitalize `webpack` as it's a proper noun
Line 34, let's avoid using `bs4` and use the `bootstrap4-theme` package name, replace `bs4 theme` with `bootstrap4-theme package`
Line 34, `bootstrap` should be capitalized.
Line 37, `slack` should be capitalized.

Thank you.